src/interactive/completions.ts

```diff
@@ -43,19 +45,25 @@ function completionItemProvider(conn: MessageConnection): vscode.CompletionItemP
                 return
             }
             const completionPromise = (async () => {
```

Comment:
Can't we write this as
```
const completionPromise = wrapCrashReporting(async () => {})()
```

Reply:
No, `wrapCrashReporting` only works with non-`async` functions.

But I am realizing that in general we need two versions of all these error hooking functions, one async one non async. I'll fix that generically in a new PR.